jest.config.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+  displayName: 'jest',
+  collectCoverageFrom: [
+    'src/**/*.js',
+    'src/**/*.jsx',
+    'src/**/*.ts',
+    'src/**/*.tsx',
+  ],
+  coverageThreshold: {
+    global: {
+      lines: 0,
+    },
+  },
+  moduleNameMapper: {
+    '\\.(css)$': 'identity-obj-proxy',
+  },
+  modulePaths: ['<rootDir>/src'],
+  reporters: [
+    'default',
+  ],
+  resetMocks: true,
+  restoreMocks: true,
+  setupFilesAfterEnv: ['<rootDir>/scripts/setupJestTests.js'],
+  testEnvironment: 'jsdom',
+  testEnvironmentOptions: {
+    url: 'http://localhost/',
+  },
+  testPathIgnorePatterns: ['.+\\.pact\\.test\\.ts$'],
```

Comment:
Is this needed? Is pact for contract testing? As we are not going to add them probably it can be removed